File: C/D1/2 - area of rect/2.cpp
// Write a C program to find the area of rectangle.

#include <stdio.h>

int main(){
    float l, b, ans;

    printf("Enter length of rectangle : ");
    scanf("%f %f", &l, &b);

    ans = l*b;
    printf("Area of rectangle : %f", ans);
}

File: Python/D2/2 max_number/2.py
# Write a python program to find out the maximum number from 2 numbers.

a = int(input('Enter the number 1 : '))
b = int(input('Enter the number 2 : '))

# if(a > b):
#     print(a, 'is max number.')
# else:
#     print(b, 'is max number.')

# Write a python program to find out the maximum number from 3 numbers.

c = int(input('Enter the number 3 : '))

if(a > b and a > c):
    print(a, 'is max number.')
elif(b > c and b > a):
    print(b, 'is max number.')
else:
    print(c, 'is max number.')